src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -284,6 +284,72 @@ For example, the path for german would be ``src/locale/messages.de-DE.xlf``.
 ``<locale-id>`` should match the id previouisly inserted in
 ``supported-languages.enum.ts``.
 
+Suggestions
+~~~~~~~~~~~
+
+Strings need to start and end in the same line as the element:
+
+.. code-block:: xml
+
+  <!-- avoid -->
+  <span i18n>
```

Comment:
How does it affect the translation negatively if the elements are used as described in the "avoid" example?

Reply:
@p-na this might answer a part of your question: https://github.com/ceph/ceph/pull/24968#discussion_r234639883

Reply:
@s0nea Thanks!